packages/apollos-ui-connected/src/HorizontalContentSeriesFeedConnected/HorizontalContentSeriesFeedConnected.js

```diff
@@ -106,7 +106,7 @@ class HorizontalContentSeriesFeedConnected extends Component {
               const connection = isParent
                 ? 'childContentItemsConnection'
                 : 'siblingContentItemsConnection';
-              const newEdges = get(fetchMoreResult.node, connection, []).edges;
+              const newEdges = get(fetchMoreResult, `node.${connection}.edges`) || []);
```

Comment:
@conrad-vanl any reason you didn't continue to rely on the `default` arg of `get? 
aka
```suggestion
              const newEdges = get(fetchMoreResult, `node.${connection}.edges`, []));
```

Reply:
because it doesn't work the same way. The `default` arg of `get` is only returned if the value is `undefined`, wheres ` || []` returns `[]` if the value is `falsey`.

If we depend on an array later on in the code, you should _not_ use the default arg of `get`.

Reply:
👍 Makes perfect sense. That nuance of the default value has bit me before.